compose/cli/main.py

```diff
@@ -268,6 +268,7 @@ def pull(self, project, options):
         Usage: pull [options] [SERVICE...]
 
         Options:
+            --skip-missing          Skip unpullable images.
```

Comment:
It'd also be to get @moxiegirl 's opinion on this wording, as I think it is a little tricky to convey clearly what we want to the user.